packages/app/src/embed/components/Sidebar/FileTree/elements.js

```diff
@@ -19,8 +19,8 @@ export const FileContainer = styled.div(props =>
 export const IconContainer = styled.span(
   css({
     display: 'inline-flex',
-    width: 16,
-    height: 16,
+    width: '16px',
```

Comment:
We have so many of these :(

Reply:
is it happening because of `styled-system` which converts `16` to `16 * ${BASE_SIZE}` ?